src/utils/queryBuilder.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash';
+import moment from 'moment';
+
+// Builds up filter string by filter type
+const eventFilters = new Map([
+  ['startDate', val => `start_date eq ${moment(val).add(1, 'day').format('YYYY-MM-DD')}`],
```

Comment:
You could interpret "start date" as the day the event starts which would reflect the current functionality.  I wouldn't mind leaving it as is for now.